harmoni_detectors/harmoni_stt/config/deepspeech_configuration.yaml

```diff
@@ -0,0 +1,9 @@
+stt:
+    default_param:
+        model_file_path: /root/harmoni_catkin_ws/model/deepspeech/models/deepspeech-0.9.3-models.pbmm
```

Comment:
Maybe you can substitute the absolute with relative path. For example:
"$ROS_WS/model/deepspeech/models/deepspeech-0.9.3-models.pbmm"

Reply:
I'll change the installation/download path for the DeepSpeech models to the harmoni_models/deepspeech directory once the TTS PR is merged.